### PR TITLE
Groundwork

### PR DESCRIPTION
**Pull request by untra**
Added a few files with instructions on how to get rethinkdb and tornado installed. Changed the readme to provide further instructions on installing rethinkdb and tornado, and added a basic "hello world" route.

Note that [rethinkdb](https://rethinkdb.com/docs/install/) has different install instructions depending on the operating system. We will likely require something like [vagrant](https://www.vagrantup.com/) to ensure that everyone's install stack is consistent. I spent more time than I'd like to admit before discovering there are separate install instructions for Linux Mint (my native os).

Although this is just an experimental branch, I'd say pull this. It's groundwork for getting started. :shaved_ice:

### CODE REVIEW

**Comment by untra** [comment]
Also I _really_ want to claim the first pull request on this project. :trollface:

**Comment by antsankov** [comment]
Damn, beat me to it. I'll merge this myself because I want to add in some of the test DB stuff I was doing with this ORM: https://github.com/linkyndy/remodel

File: app.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import tornado.options
+import tornado.web
+import rethinkdb as db
+from tornado import ioloop, gen
+from tornado.concurrent import Future, chain_future
+from tornado.options import define, options
+
+#
+db.set_loop_type("tornado")
+connection = db.connect(host='localhost', port=28015)
```
Comment (antsankov) [inline app.py:13]: I was using this interesting [ORM](https://github.com/linkyndy/remodel) to handle all of the boilerplate connection, table creation, etc. Maybe we can connect to the database manually for async or complex queries that the ORM can't handle?

**Comment by nicot** [comment]
Merge it! Alex's patch addresses all of his comments, and suggestions that I would make.

**Pull request merged**